core/storage/suggestion/gae_models.py

```diff
@@ -233,20 +233,24 @@ def create(
         Raises:
             Exception. There is already a suggestion with the given id.
         """
-        instance_id = thread_id
-
-        if cls.get_by_id(instance_id):
+        if cls.get_by_id(thread_id):
             raise Exception(
                 'There is already a suggestion with the given'
-                ' id: %s' % instance_id)
+                ' id: %s' % thread_id)
 
         cls(
-            id=instance_id, suggestion_type=suggestion_type,
-            target_type=target_type, target_id=target_id,
+            id=thread_id,
+            suggestion_type=suggestion_type,
+            target_type=target_type,
+            target_id=target_id,
             target_version_at_submission=target_version_at_submission,
-            status=status, author_id=author_id,
-            final_reviewer_id=final_reviewer_id, change_cmd=change_cmd,
-            score_category=score_category, language_code=language_code).put()
+            status=status,
+            author_id=author_id,
+            final_reviewer_id=final_reviewer_id,
+            change_cmd=change_cmd,
+            score_category=score_category,
+            language_code=language_code
+        ).put_for_human()
```

Comment:
Why is this one a ``put_for_human``?

Reply:
Right, changed to `put_depending_on_id`.